provider/aws/lambda/cfevents/main.go

```diff
@@ -155,3 +133,18 @@ func parseMessage(msg string) (Message, error) {
 
 	return m, nil
 }
+
+func getLogGroup(stack string) (string, error) {
+	res, err := cf.DescribeStackResources(&cloudformation.DescribeStackResourcesInput{
```

Comment:
@ddollar @nzoschke Haven't used this myself, but this kind of sounds like what the [blox cluster-state-service](https://github.com/blox/blox/tree/master/cluster-state-service) they announced at reinvent was meant for. Maybe too much for what you're trying to achieve here.